lib/virt_autotest/utils.pm

```diff
@@ -273,4 +282,34 @@ sub remove_additional_nic {
     return script_run($cmd);
 }
 
+sub collect_virt_system_logs {
+    if (script_run("test -f /var/log/libvirt/libvirtd.log") == 0) {
+        upload_logs("/var/log/libvirt/libvirtd.log");
+    } else {
+        record_info "File /var/log/libvirt/libvirtd.log does not exist.";
+    }
+
+    if (script_run("test -d /var/log/libvirt/libxl/") == 0) {
+        assert_script_run 'tar czvf /tmp/libxl.tar.gz /var/log/libvirt/libxl/';
+        upload_asset '/tmp/libxl.tar.gz';
+    } else {
+        record_info "Directory /var/log/libvirt/libxl/ does not exist.";
+    }
+
+    if (script_run("test -d /var/log/xen/") == 0) {
+        assert_script_run 'tar czvf /tmp/xen.tar.gz /var/log/xen/';
+        upload_asset '/tmp/xen.tar.gz';
+    } else {
+        record_info "Directory /var/log/xen/ does not exist.";
+    }
+
+    assert_script_run 'mkdir /tmp/dumpxml';
```

Comment:
Would make it `mkdir -p /tmp/dumpxml` because `mkdir /tmp/dumpxml` would fail if the directory already exists